### DISCUSSION TITLE
Dedicated Notes Function(s)

### DISCUSSION THREAD

**Discussion by sirinath** [typst/typst#1335] [Ideas]
Can the following be added and documented:

- Sidenotes
- Marginal notes
- Footnotes
- Endnotes
- Chapter / Section notes
- Column footnotes
- etc.

**Comment by PgBiel** [comment]
Footnotes were added in v0.4.0 (`#footnote[...]`). The rest are probably possible to implement in libraries, but make for interesting ideas. Perhaps open a feature request issue for them?
